legend-engine-xts-data-space/legend-engine-xt-data-space-generation/src/main/java/org/finos/legend/engine/generation/analytics/model/DataSpaceExecutableAnalysisResult.java

```diff
@@ -23,7 +23,7 @@ public class DataSpaceExecutableAnalysisResult
     public String title;
     public String description;
     public String executable;
-    public DataSpaceExecutableInfo info;
+    public List<DataSpaceExecutableInfo> infoList;
```

Comment:
This is a breaking change, I think the spirit is to always assume only the first execution context